base/complex.jl

```diff
@@ -272,6 +275,10 @@ muladd(z::Complex, w::Complex, x::Complex) =
     Complex(muladd(real(z), real(w), real(x)) - imag(z)*imag(w), # TODO: use mulsub given #15985
             muladd(real(z), imag(w), muladd(imag(z), real(w), imag(x))))
 
+#unary + handling Complex{Bool}
+
++(z::Complex{Bool}) = Complex(convert(Int,z))
```

Comment:
A nice way to write this would be `Complex(+z.re, +z.im)` so that it just delegates to whatever real unary `+` does.

Reply:
Ok!